README.md

```diff
@@ -43,7 +43,16 @@ This creates:
 * a Subcription to the OCS catalog in the `openshift-storage`
 namespace
 
-Once this is done, a StorageCluster can be created.
+You can check the status of CSV by command `oc get csv -n openshift-storage`
+The output should look like:
+
+```console
+NAME                  DISPLAY                                VERSION   REPLACES   PHASE
+ocs-operator.v0.0.1   Openshift Container Storage Operator   0.0.1                Succeeded
+```
+
+This can takes few minutes and once PHASE is Succeeded you can create
```

Comment:
Change the above to: ``This can take a few minutes. Once PHASE says `Succeeded` you can create``.

Reply:
done